src/functions/durableEntity.ts

```diff
@@ -0,0 +1,47 @@
+import { app, HttpHandler, HttpRequest, HttpResponseBody, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { EntityHandler } from 'durable-functions';
+
+// Replace with your own Durable entity name
+const entityName = 'Counter';
+
+const clientInput = df.input.client();
```

Comment:
In terms of naming, I'd have an easier time reading this if this contained the term `durableClient` in the name.
Some options: `durableClient`, `durableClientBinding`, `durableClientInput`, etc.

Reply:
Renamed to `durableClient`